### PR TITLE
Tesla: more fingerprint for AP1

### PR DESCRIPTION
**Pull request by kennedyshead**
**Description** In rare occasions my AP1 was not recognised, this fixes the problem.

**Verification** tested in all different states I could imagine the car being in

### CODE REVIEW

File: selfdrive/car/tesla/values.py [hunk 1]
```diff
@@ -21,6 +21,9 @@ class CAR:
...
     },
+    {
```
Comment (adeebshihadeh) [inline selfdrive/car/tesla/values.py:24]: this is just a subset of the original fingerprint, but 660 and 857 have different lengths

Comment (kennedyshead) [reply selfdrive/car/tesla/values.py:24]: Nope, I saw that and I have tried to figure out why that is for a week now. It might be a EU version?!

Comment (adeebshihadeh) [reply selfdrive/car/tesla/values.py:24]: are these values taken from another car besides your own?

Comment (kennedyshead) [reply selfdrive/car/tesla/values.py:24]: No these new ones are from mine, the ones before where borrowed from tinkla

Comment (kennedyshead) [reply selfdrive/car/tesla/values.py:24]: should I just replace?

Comment (adeebshihadeh) [reply selfdrive/car/tesla/values.py:24]: yeah, let's just replace with yours

**Pull request merged**